identity/peer.go

```diff
@@ -0,0 +1,30 @@
+package identity
+
+import (
+	"fmt"
+
+	"perun.network/go-perun/wallet"
+)
+
+type NetworkAdapter string
+
+const (
+	TCPIP NetworkAdapter = "tcpip"
+)
+
+type URL struct {
+	IPAddr   string
+	Port     uint16
+	Endpoint string
```

Comment:
This is not used right now?

Reply:
It is used, but after thinking through i feel this struct may not handle all types of network address that we want to use. Since the scope of this issue is only to setup the client, i have updated the implementation to use string data type for URL. 

Later during the implementation of Identity Management module, we can define the requirements and implement the data type to handle URLs.